index.html

```diff
@@ -27,13 +27,13 @@ <h1 class="first-title">I'm Omar Glad to see You</h1>
       </p>
     </section>
     <section class="section-2">
-      <p class="connect">LET'S CONNECT</p>
+      <h3 class="connect">LET'S CONNECT</h3>
       <div class="container-2">
-        <img src="images/twiter.svg" alt="" />
-        <img src="images/Linkedin.svg" alt="" />
-        <img src="images/m.svg" alt="" />
-        <img src="images/github.svg" alt="" />
-        <img src="images/manito.svg" alt="" />
+        <img src="images/twiter.svg" alt="Twitter" />
+        <img src="images/Linkedin.svg" alt="Linked-in" />
+        <img src="images/m.svg" alt="M" />
+        <img src="images/github.svg" alt="GitHub" />
+        <img src="images/manito.svg" alt="Manito" />
       </div>
```

Comment:
- [ ] The requirements say that The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag)